file-share-service-desktop-client/Modules/Admin/JobViewModels/NewBatchJobViewModel.cs

```diff
@@ -446,7 +457,9 @@ public class NewBatchFilesViewModel
         public string MimeType => newBatchFile.MimeType;
 
         public bool CorrectNumberOfFilesFound => ExpectedFileCount == Files.Count();
-        
+        public IEnumerable<KeyValuePair<string, string>> Attributes =>
+        //newBatchFile.Attributes.Select(kv => new KeyValuePair<string, string>(kv.Key, ExpandMacros(kv.Value)));
```

Comment:
and delete this commented line?

Reply:
Macros related changes will be taken care in the PBI 11863, so removing this commented line.

Reply:
11863 has already been merged. Please choose one of these lines and remove the other

Reply:
updated using macros. Thanks